xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -348,13 +348,13 @@ bool CPVRChannelGroupInternal::UpdateGroupEntries(const CPVRChannelGroup &channe
   return bReturn;
 }
 
-void CPVRChannelGroupInternal::CreateChannelEpg(CPVRChannelPtr channel, bool bForce /* = false */)
+void CPVRChannelGroupInternal::CreateChannelEpg(CPVRChannelPtr channel)
```

Comment:
Again the "by value" vs. "by ref" thing. CPVRChannelPtr => const CPVRChannelPtr & would be better.